lib/index.ts

```diff
@@ -1,22 +1,65 @@
-import 'source-map-support/register';
-import {DepGraph, DepGraphBuilder, PkgInfo} from "@snyk/dep-graph";
+import "source-map-support/register";
+import { DepGraph, DepGraphBuilder, PkgInfo } from "@snyk/dep-graph";
 import * as os from "os";
 
-export function parseClojureDeps(name: string, version: string, depsEdnContent: string): DepGraph {
-  const rootPackage: PkgInfo = {name, version};
-  const depGraph = new DepGraphBuilder({name: 'clojure-deps'}, rootPackage);
+export function parseLein(
+  name: string,
+  version: string,
+  leinDepsTreeContent: string
+): DepGraph {
+  const parseLine = (line: string) =>
```

Comment:
In general, it's better to use `function parseLine (line: string) {}` syntax, mainly because of better stack-traces. And it will be nicer for SAST tools analysing your code :-D 

You can also pass the function by it's name as a parameter.

But it's not a big deal here.